scripts/deploy_local.sh

```diff
@@ -249,15 +270,11 @@ write_envs_config() {
   rm ${SOURCE_DIR}/gs_env || true
```

Comment:
since you delete a file, please print a warning... e.g., 
`Found gs_env exists, remove the environment variables and generate a new file.`